extensions/vtk/src/OHIFVTKViewport.js

```diff
@@ -267,11 +267,13 @@ class OHIFVTKViewport extends Component {
     const spacing = vtkImageData.getSpacing();
     // Set the sample distance to half the mean length of one side. This is where the divide by 6 comes from.
     // https://github.com/Kitware/VTK/blob/6b559c65bb90614fb02eb6d1b9e3f0fca3fe4b0b/Rendering/VolumeOpenGL2/vtkSmartVolumeMapper.cxx#L344
-    const sampleDistance = (spacing[0] + spacing[1] + spacing[2]) / 6;
+    const sampleDistance =
+      (parseFloat(spacing[0]) + parseFloat(spacing[1]) + parseFloat(spacing[2])) /
```

Comment:
I think this should be a metadata provider change and not specifically to VTK?

Reply:
hmmm, it appears to be coming from here
https://github.com/OHIF/react-vtkjs-viewport/blob/HEAD/src/lib/data/buildMetadata.js#L3

Reply:
This function uses the pixel module, you just need to go to metadata provider in OHIF core and parse the data there (pixel spacing) and call it a day

Reply:
VTK extension doesn't get its metadata from OHIF metadataProvider, it gets it from react-vtkjs-viewport

Reply:
I'm looking at it right now

Reply:
If it's using a module from cornerstone then it's using the metadata provider which is fed by OHIF (given the DICOM files fetched) and passed to cornerstone